package.json

```diff
@@ -8,21 +8,25 @@
   "scripts": {
     "bootstrap": "lerna bootstrap --no-ci",
     "build": "lerna run build",
+    "prettier": "lerna run prettier",
+    "prettier.dry-run": "lerna run prettier.dry-run",
     "test": "lerna run test",
     "changelog": "lerna-changelog"
   },
   "devDependencies": {
+    "@ionic/prettier-config": "^2.0.0",
     "@stencil/core": "^2.9.0",
     "@types/jest": "^26.0.9",
     "jest": "^26.3.0",
     "lerna": "^4.0.0",
     "lerna-changelog": "^1.0.1",
     "np": "^6.0.2",
-    "prettier": "^2.0.5",
+    "prettier": "2.7.1",
     "rimraf": "^2.6.3",
     "rollup": "^2.23.1",
+    "semver": "^5.5.0",
```

Comment:
This moved as a result of adding new dependencies via `npm`'s CLI - might as well keep it 🤷